test_scripts/API/VehicleData/SubscribeVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua

```diff
@@ -0,0 +1,54 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed but parameter of this request is NOT allowed by Policies
+--
```

Comment:
@AByzhynar, add link to the requirement when they will be available.

Reply:
@AStasiuk Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45